appserver/website-bootstrap/main.css

```diff
@@ -201,6 +203,8 @@ body {
 #svg {
```

Comment:
this item in the css is referring to an HTML element with 'id' attribute "svg", right? If the html now doesn't contain such an element, but instead contains an element with 'id' attribute "mapsvg", should this line be changed to 
#mapsvg {
?